Source/Public/Remove-VSTeamTaskGroup.ps1

```diff
@@ -0,0 +1,27 @@
+function Remove-VSTeamTaskGroup {
+   [CmdletBinding()]
```

Comment:
Add SupportsShouldProcess = $true and a  ConfirmImpact value. You can review Remove-VSTeamBuildTag as an example.